### PR TITLE
WX-859 Add config for workflow-execution-identity

### PR DESCRIPTION
**Pull request by kraefrei**

### CODE REVIEW

File: coa-helm/templates/cromwell.yaml [hunk 1]
```diff
@@ -141,6 +141,7 @@ data:
             endpoint = "http://tes/v1/tasks"
             use_tes_11_preview_backend_parameters = true
             transform-blob-to-local-path = true
+	    workflow-execution-identity = "{{ .Values.config.workflowExecutionIdentity }}"
```
Comment (jgainerdewar) [inline coa-helm/templates/cromwell.yaml:144]: Haha, I think Github is yelling at you about a tab character here. 

We'll also want to update `values.yaml` and `local_values.template.yaml` in this chart.

**Comment by jgainerdewar** [comment]
FYI @rtitle

**Review by jgainerdewar — APPROVED** [review]

**Review by rtitle — APPROVED** [review]
Thanks, 👍 

Leo ticket to update: https://broadworkbench.atlassian.net/browse/TOAZ-303

File: coa-helm/values.yaml [hunk 2]
```diff
@@ -12,6 +12,7 @@
   dockerInDockerImageName: ""
   usePreemptibleVmsOnly: false
   drsUrl: RUNTIME_PARAMETER
+  workflowExecutionIdentity: RUNTIME_PARAMETER
```
Comment (rtitle) [inline coa-helm/values.yaml:15]: FWIW, I _think_ this is the same value as `identity.resourceId` (L37) which Leo is already setting -- but updating Leo to send `config.workflowExecutionIdentity` as well.

**Pull request merged**